scripts/run_e2e_tests.sh

```diff
@@ -251,3 +251,6 @@ else
     node_modules/protractor/bin/protractor core/tests/protractor-browserstack.conf.js --capabilities.shardTestFiles="$SHARDING" --capabilities.maxInstances=$SHARD_INSTANCES --suite "$SUITE" --params.devMode="$DEV_MODE"
   fi
 fi
+
+# Print log from backend server.
+cat log.txt
```

Comment:
Should probably echo "Printing server log" before this.

Also, should we delete the file thereafter?

Reply:
The log isn't really necessary. We should remove capturing it altogether.

Reply:
Removed.